app/controllers/api/v1/assessments_controller.rb

```diff
@@ -0,0 +1,165 @@
+class Api::V1::AssessmentsController < Api::V1::BaseApiController
+
+  include AssessmentHandinCore
+  include AssessmentAutogradeCore
+
+  before_action -> {doorkeeper_authorize! :user_courses}, only: [:index, :problems, :writeup, :handout]
+  before_action -> {doorkeeper_authorize! :user_submit}, only: [:submit]
+
+  before_action :set_assessment, except: [:index]
+
+  def index
+    asmts = @course.assessments.ordered
+    allowed = [:name, :display_name, :description, :start_at, :due_at, :end_at, :updated_at, :max_grace_days, :handout, :writeup, :max_submissions, :disable_handins, :category_name, :group_size, :has_scoreboard, :has_autograder]
```

Comment:
Don't think it's necessary for `has_scoreboard` and `has_autograder` to be returned for the student